data/fixtures/agents.json

```diff
@@ -0,0 +1,110 @@
+[
+    {
+        "model": "tr_ars.agent",
+        "pk": 10,
+        "fields": {
+            "name": "ara-arax-agent",
+            "description": null,
+            "uri": "http://localhost:8000/arax/api",
+            "contact": null,
+            "registered": "2020-08-04T18:56:34.339Z",
+            "updated": "2020-08-04T18:56:34.340Z"
+        }
+    },
+    {
+        "model": "tr_ars.agent",
+        "pk": 20,
+        "fields": {
+            "name": "ara-bte-agent",
+            "description": null,
+            "uri": "http://localhost:8000/bte/api",
+            "contact": null,
+            "registered": "2020-08-03T19:12:14.721Z",
+            "updated": "2020-08-03T19:12:14.722Z"
+        }
+    },
+    {
+        "model": "tr_ars.agent",
+        "pk": 30,
+        "fields": {
+            "name": "ara-robokop-agent",
+            "description": "simple wrapper around robokop agent",
+            "uri": "http://localhost:8000/robokop/api",
+            "contact": "hello@ncats.io",
+            "registered": "2020-05-13T13:07:47Z",
+            "updated": "2020-07-09T16:35:53.049Z"
+        }
+    },
+    {
+        "model": "tr_ars.agent",
+        "pk": 40,
+        "fields": {
+            "name": "ara-rtx-agent",
+            "description": null,
+            "uri": "http://localhost:5000",
```

Comment:
It is wierd to see localhost specified here ... but even in production, this should work, right?

Reply:
And these port numbers are immutable, right?